uiowa/uiowa
# Feature: Timeline

Resolves https://github.com/uiowa/uiowa/issues/3314. 
Resolves https://github.com/uiowa/uiowa/issues/3951.

- [x] Admissions split
- [x] Permissions for webmasters and editors need to be added across all v3 splits
- [x] Can the fontawesome field be moved down? As an optional field, it feels stuffy at the top.
- [x] Restrict additional font awesome settings
- [x] Headline field doesn't seem to link up to children.

# How to test
```
blt ds --site=sandbox.uiowa.edu
yarn workspace uids_base gulp --development
drush @sandbox.local cr
```

- Add a "Timeline" block to a one column layout (the block is/should be restricted to one column layouts only).
- Add a number of timeline items and test out the fields
- User stories:
 -  "As a site owner, I can build a timeline layout with layout builder"
 - As an admin, I can use the additional font awesome settings on the timeline icon field
 - As an webmaster, I can see the additional font awesome settings on the timeline icon field but they are disabled for use

## Also
- Added access to People block list for Grad, Transportation, and Admissions which previously did not have the people list block.
